views/celebrities/celebrities.hbs

```diff
@@ -0,0 +1,10 @@
+<h2>All Celebrities</h2>
+{{#each celebs}}
+<h3>{{name}}</h3>
+<h4>{{occupation}}</h4>
+<p>{{catchphrase}}</p>
+
+<a href="/celebrities/{{_id}}/edit">Edit Celebrity</a>
```

Comment:
The route `/celebrities/{{_id}}/edit` doesn't exist yet, so this `<a>` tag is taking you nowhere!